src/ActionModal/components/ModalPropose.vue

```diff
@@ -249,7 +249,7 @@ export default {
           address: this.userAddress,
           denom: fromMicroDenom(this.denom)
```

Comment:
this feels wrong. the API should imo serve the correct values. comes back to your point about maybe standardizing the transactions